src/algebra/big_operators/basic.lean

```diff
@@ -1220,6 +1220,16 @@ lemma prod_pow_boole [decidable_eq α] (s : finset α) (f : α → β) (a : α)
   (∏ x in s, (f x)^(ite (a = x) 1 0)) = ite (a ∈ s) (f a) 1 :=
 by simp
 
+lemma prod_dvd_prod {S : finset α} (g1 g2 : α → β) (h : ∀ a ∈ S, g1 a ∣ g2 a) :
```

Comment:
This is true more generally for multiset, right?

Reply:
You're right, thanks for pointing this out.  I've proved this and the lemma you suggested in your other comment in a separate PR #11693.